### PR TITLE
Contributing Doc Updates

### PR DESCRIPTION
**Pull request by lalli-flores**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -17,23 +17,38 @@ If you would like to contribute to this repository by adding features, enhanceme
...
+    2. Wait for Github to finish forking the repository
+    3. Head over to your newly created repository fork
+
+2. Clone your repository fork locally
+
+    ```bash
+    git clone git@github.com:{your username}/components.git
+    ```
+
+3. Install all project dependencies
```
Comment (adamweeks) [inline CONTRIBUTING.md:44]: Does `npm-install-peers` do a full install as well?

Comment (lalli-flores) [reply CONTRIBUTING.md:44]: Yes, install peers + others

Comment (adamweeks) [reply CONTRIBUTING.md:44]: Could we change this to 
```suggestion
3. Install all project dependencies and peers
```
to avoid confusion?

**Review by adamweeks — APPROVED** [review]
One minor comment, but non blocking. Thanks for updating this, it will be helpful for other contributors!

File: CONTRIBUTING.md (hunk 1 above)
Comment (lalli-flores) [reply CONTRIBUTING.md]: Updated

**Pull request merged**